contracts/DCAStrategies/DCAStrategies/DCAStrategiesManagementHandler.sol

```diff
@@ -4,18 +4,45 @@ pragma solidity >=0.8.7 <0.9.0;
 import '../../interfaces/IDCAStrategies.sol';
 
 abstract contract DCAStrategiesManagementHandler is IDCAStrategiesManagementHandler {
+  mapping(uint80 => Strategy) internal _strategies;
+  mapping(string => uint80) internal _strategyNames;
+  mapping(bytes32 => IDCAStrategies.ShareOfToken) internal _tokenShares;
+  uint80 public strategyCounter; // NOTE: could we use _mintCounter from DCAStrategiesPermissionsHandler?
```

Comment:
They are different things. `_mintCounter` will count positions, while this counter will count strategies. So it's ok that we have this

Reply:
You are right! I don't know how I didn't realize it